Atomic/util.py

```diff
@@ -235,14 +235,16 @@ def skopeo_layers(image, args=[], layers=[]):
     temp_dir = tempfile.mkdtemp()
     try:
         args = ['skopeo', 'layers'] + args + [image] + layers
-        r = subp(args, cwd=temp_dir)
-        if r.return_code != 0:
-            check_v1_registry(image)
-            raise ValueError("Unable to interact with this registry: {}".format(r.stderr))
-    except OSError:
-        raise ValueError("skopeo must be installed to perform remote inspections")
-    finally:
+        try:
+            r = subp(args, cwd=temp_dir)
+            if r.return_code != 0:
+                check_v1_registry(image)
+                raise ValueError("Unable to interact with this registry: {}".format(r.stderr))
+        except OSError:
+            raise ValueError("skopeo must be installed to perform remote inspections")
+    except Exception as e:
         shutil.rmtree(temp_dir)
```

Comment:
Can  you be more specific on the Exceptions (ValureError)?  pylint does not like this.    Do you want to remove the layers on success?  This looks like it was before, is that the intended change?